index.html

```diff
@@ -0,0 +1,70 @@
+          fill="#6070FF" />
+      </svg>
+    </a>
+    <nav>
+      <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path fill-rule="evenodd" clip-rule="evenodd"
+          d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z"
+          fill="#6070FF" />
+      </svg>
+    </nav>
+  </header>
+
+  <main class="container">
+    <section class="main section">
+
+      <section class="headline-section">
+        <Section class="headline-content">
+
+          <h2 class="heading1">I’m Mariane Glad to see you!!</h2>
+
+          <p class="paragraph1">I’m a software developer! I can help you build a product , feature or website
+            Look
+            through
+            some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h2 class="lets-connect">LET’S CONNECT</h2>
+          <footer>
+            <ul class="social-media">
+              <li><a href="#"><img src="./images/twitter.png" alt="twitter-icon"></a></li>
+              <li><a href="#"><img src="./images/linkedin.png" alt="linkedin-icon"></a></li>
+              <li><a href="#"><img src="./images/mail.png" alt="mail-icon"></a></li>
+              <li><a href="#"><img src="./images/github.png" alt="github-icon"></a>
+              <li>
+              <li><a href="#"><img src="./images/fingers.png" alt="fingers-icon"></a>
+              <li>
+
+
```

Comment:
- [ ] Please remove unnecessary codes for example this `<li>` has no content inside and It looks like it is not playing any role in our HTML. That is why I would suggest that you remove it. In addition, you may consider removing unnecessary white spaces in your codes as they make it difficult to read and understand your code 😃 👍